intelmq/conf/harmonization.conf

```diff
@@ -1,226 +1,357 @@
+            "type": "FQDN"
+        },
+        "destination.tor_node": {
+            "description": "If the destination IP was a known tor node.",
+            "type": "Boolean"
+        },
+        "destination.url": {
+            "description": "A URL denotes on IOC, which refers to a malicious resource, whose interpretation is defined by the abuse type. A URL with the abuse type phishing refers to a phishing resource.",
+            "type": "URL"
+        },
+        "event_description.target": {
+            "description": "Some sources denominate the target (organization) of a an attack.",
+            "type": "String"
+        },
+        "event_description.text": {
+            "ascii": true,
+            "description": "A free-form textual description of an abuse event.",
+            "type": "String"
+        },
+        "event_description.url": {
+            "description": "A description URL is a link to a further description of the the abuse event in question.",
+            "type": "URL"
+        },
+        "event_hash": {
+            "description": "Computed event hash with specific keys and values that identify a unique event. At present, the hash should default to using the SHA1 function. Please note that for an event hash to be able to match more than one event (deduplication) the receiver of an event should calculate it based on a minimal set of keys and values present in the event. Using for example the observation time in the calculation will most likely render the checksum useless for deduplication purposes.",
+            "regex": "^[a-fA-F0-9]+$",
+            "type": "String"
+        },
+        "feed.code": {
+            "description": "Code name for the feed, e.g.  DFGS, HSDAG etc.",
+            "length": 100,
+            "type": "String"
+        },
+        "feed.name": {
+            "description": "Name for the feed, usually found in collector bot configuration.",
+            "type": "String"
+        },
+        "feed.url": {
+            "description": "The URL of a given abuse feed, where applicable",
+            "type": "URL"
+        },
+        "malware.hash": {
+            "description": "A string depicting a checksum for a file, be it a malware sample for example.",
+            "regex": "^[a-fA-F0-9]+$",
+            "type": "String"
+        },
+        "malware.name": {
+            "description": "A malware family name in lower case.",
+            "regex": "[a-z ]+",
+            "type": "MalwareName"
+        },
+        "malware.version": {
+            "ascii": true,
+            "description": "A version string for an identified artifact generation, e.g. a crime-ware kit.",
+            "type": "String"
+        },
+        "misp_ip": {
```

Comment:
so, we will not use 'extra' field ?

btw, change to misp_uuid as mentioned in #280